packages/blockchain-wallet-v4-frontend/src/data/cache/reducers.js

```diff
@@ -31,6 +31,9 @@ const cache = (state = INITIAL_STATE, action) => {
       const { phonePubkey } = payload
       return assoc('channelPhonePubkey', phonePubkey, state)
     }
+    case AT.DISCONNECT_CHANNEL_PHONE: {
+      return assoc('channelPhonePubkey', undefined, state)
```

Comment:
We also need to call this when the user logs out and selects de auth browser.
<img width="553" alt="Screen Shot 2020-09-24 at 2 44 04 PM" src="https://user-images.githubusercontent.com/5640772/94146574-6ad18980-fe74-11ea-9432-62719a384b39.png">

Reply:
@plondon 👍

Reply:
done

Reply:
Where?

Reply:
@plondon here https://github.com/blockchain/blockchain-wallet-v4-frontend/blob/aa0a3accff99d7eb6aafe12e20893595cc557648/packages/blockchain-wallet-v4-frontend/src/data/auth/sagas.js#L504 in de-authorise saga.

Reply:
Just to make sure this doesn't get lost. 
When the user logs in with a *new* GUID, i.e. because he has multiple wallets, we also want the browser to forget the phone identity. This is because we don't want a complicated list of mappings in the browser to select from.